File: 16000/BOJ_16466 콘서트.py
# https://www.acmicpc.net/problem/16466
global arr

def func():
    for i in range(1, 2200000000):
        if i-1 == len(arr) or i < arr[i-1]:
            return i

n = int(input())
arr = list(map(int, input().split(' ')))
arr.sort()

print(func())

File: 16000/BOJ_16466 콘서트.cpp
// https://www.acmicpc.net/problem/16466
#include <iostream>
#include <algorithm>
using namespace std;
int n, arr[1000000];

int main()
{
    cin >> n;

    for(int i = 0; i < n; ++i)
        cin >> arr[i];
    
    sort(arr, arr+n);
    int answer = 1;
    for(int i : arr)
        if(answer == i) ++answer;

    cout << answer;
}